cmd/flow-aggregator/options.go

```diff
@@ -90,27 +92,37 @@ func (o *Options) validate(args []string) error {
 			return fmt.Errorf("IPFIX flow collector is given in invalid format: %v", err)
 		}
 		if proto == "udp" {
-			o.flowCollectorAddr, err = net.ResolveUDPAddr("udp", hostPortAddr)
+			o.externalFlowCollectorAddr, err = net.ResolveUDPAddr("udp", hostPortAddr)
 			if err != nil {
 				return fmt.Errorf("IPFIX flow collector over UDP proto cannot be resolved: %v", err)
 			}
 		} else {
-			o.flowCollectorAddr, err = net.ResolveTCPAddr("tcp", hostPortAddr)
+			o.externalFlowCollectorAddr, err = net.ResolveTCPAddr("tcp", hostPortAddr)
 			if err != nil {
 				return fmt.Errorf("IPFIX flow collector over TCP proto cannot be resolved: %v", err)
 			}
 		}
 	}
-	if o.config.FlowExportInterval != "" {
+	if o.config.FlowExportInterval == "" {
+		o.exportInterval, _ = time.ParseDuration("60s")
```

Comment:
60s can be defined as constant with default prefix.

Reply:
Thanks, addressed.